packages/cli/src/__tests__/ceramic-daemon.test.ts

```diff
@@ -290,6 +290,27 @@ describe('Ceramic interop: core <> http-client', () => {
         expect(StreamUtils.serializeState(docV5Core.state)).toEqual(StreamUtils.serializeState(docV5Client.state))
     })
 
+    it('can get stream contents from /raw_data', async () => {
+      const content1 = {test: 123}
+      const content2 = {test: 456, test2: 'abc'}
+      const content3 = {test2: 'def'}
+      const doc = await TileDocument.create(core, content1)
+
+      await anchorDoc(doc)
```

Comment:
Since you're not testing anything related to anchoring in this test, you can remove the calls to anchorDoc and add `{ anchor: false }` to the `create` and `update` calls, which will speed up the test

Reply:
I tried [adding `anchor: false` to the various calls](https://github.com/ceramicnetwork/js-ceramic/pull/1395/files#diff-077b6994383a23f7d2210366626822554ebbcde4d9de8e8a5f94a93a7d1b1141R298-R300) and the test fails as through the updates are not applied:

```
● Ceramic interop: core <> http-client › can get stream contents from /raw_data

    expect(received).toEqual(expected) // deep equality

    - Expected  - 1
    + Received  + 1

      Object {
    -   "test2": "def",
    +   "test": 123,
      }
```

Reply:
Most likely because the current version lives in `state.next.content` not in `state.content`.  If you switch the raw_data endpoint to use the new `rawData` accessor you added to the Stream class, that should fix it.

Reply:
Done. Tests are passing now.